lib/kube/proxy/moderated_sessions_test.go

```diff
@@ -0,0 +1,425 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
Is this correct? Is this kube code or Teleport code?

Reply:
I copied the wrong header